README.md

```diff
@@ -31,6 +31,7 @@ class { 'firewalld': }
 * `service_ensure`: Whether the service should be running or not (default: running)
 * `service_enable`: Whether to enable the service
 * `default_zone`: Optional, set the default zone for interfaces (default: undef)
+* `firewallbackend`: Optional, set the firewall backend for firewalld (default: undef)
```

Comment:
Suggest changing to `firewall_backend` to match the current pattern and read easier.